content/applications/general/companies.rst

```diff
@@ -4,89 +4,196 @@
 Companies
 =========
 
-A centralized management environment allows you to select multiple companies simultaneously and set
-their specific warehouses, customers, equipment, and contacts. It provides you the ability to
-generate reports of aggregated figures without switching interfaces, which facilitates daily tasks
-and the overall management process.
+A centralized management environment allows an administrator to select multiple companies
+simultaneously, and set their specific warehouses, customers, equipment, and contacts. It provides
+the ability to generate reports of aggregated figures without switching interfaces, which
+facilitates daily tasks, and enhances the overall management process.
+
+.. danger::
+   Utilizing a multi-company environment in an Odoo database warrants a subscription change from
+   *Standard* (or *One App Free*) to a *Custom* plan. Should the administrator not migrate the
+   subscription to a *Custom* plan after adding another company, the database risks being
+   deactivated. For more information, consult the `Odoo Pricing page
+   <https://www.odoo.com/pricing-plan>`_.
+
+   If the customer is on the *Standard* plan, and adds multi-company functionality to the database,
+   it triggers an upsell for the *Custom* plan.
+
+   If they are on a *Yearly* or *Multi-year* contract, this creates an upsell order, with a 30-day
+   limit (risking deactivation). If multi-company functionality is fully removed within that time
+   period, the next time the database pings, the upsell order and limit are removed automatically.
+
+   If the customer is on a *Monthly* contract, once the next bill is created, and multi-company
+   functionality is still detected while they are on a *Standard* plan, they are automatically
+   switched, and billed on the *Custom* plan.
+
+   Should the customer want to go back to *Standard*, the Customer Success Team **must** be
+   contacted.
+
+To create a new company, navigate to :menuselection:`Settings app --> Companies section`, and click
+:guilabel:`Manage Companies`. Then, click :guilabel:`New` to create a new company.
+
+Proceed to fill out the new company form that appears.
+
+.. tip::
+   To archive a company, navigate to :menuselection:`Settings app --> Companies section --> Manage
+   Companies`. Then, tick the checkbox to the left of the company to be archived. If the
+   :guilabel:`Companies` page is not in list view, click the :guilabel:`≣ (four bars)` icon, located
+   in the top-right corner of the page.
+
+   After selecting the appropriate company, click the :guilabel:`Action ⚙️ (gear)` icon, and select
```

Comment:
On the UI the button is labeled "⚙️ Actions" and not "Action ⚙️", no need for icon descriptor when an icon is accompanied by a label :)
```suggestion
   After selecting the appropriate company, click the :guilabel:`⚙️ Actions` icon, and select
```